### ISSUE TITLE
Questions about the BN layer and affine layers

### ISSUE THREAD

**Issue by gooners1886** [facebookresearch/video-nonlocal-net#34]
In the readme file: 
To train the i3d Non-local Networks with longer clips (32-frame input), we first need to obtain the model trained from "run_i3d_baseline_400k.sh" as a pre-trained model. Then we convert the Batch Normalization layers into Affine layers by running:  python modify_caffe2_ftvideo.py  xxxx

what is an affine layer?  Is it a conv layer without batch normalizaion?  But I find the model with affine layer has more parameters than the one with BN layer. Is there some other operation?

**Comment by bryanyzhu** [comment]
+1
